docs/UI/SetGlobalProperties/index.md

```diff
@@ -9,30 +9,98 @@ Sender
+- display the onscreen keyboard upon User\`s request within the following condition: all keyboardProperties supported by HMI must be embodied in HMI_capabilities.json file. In this case SDL are able to compare keyboardProperties requested by mobile device with actual supported keyboardProperties and send to HMI only that are supported.    
+- use default keyboardProperties – parameter in case SDL transfers UI.SetGlobalProperties request with omitted or empty keyboardProperties param to HMI.   
+_**Important Note:**_   
+_If HMI-defined VR commands are accessible together with those provided by SDL via VR.AddCommand, HMI must:_ 
+- _add the corresponding VR HMI-defined commands to the list of VR help items provided by SDL via UI.SetGlobalProperties_
+- _display the complete list of available VR commands (SDL-defined and HMI-defined ones) when the User activates VR._    
+
+**3.** Respond to the request.   
+
+
 ### Request
 
 #### Parameters
 
 |Name|Type|Mandatory|Additional|
 |:---|:---|:--------|:---------|
 |vrHelpTitle|String|false|maxlength: 500|
-|vrHelp|[Common.VrHelpItem](../../common/structs/#vrhelpitem)|false|array: true<br>minsize: 1<br>maxsize: 100|
+|vrHelp|[Common.VrHelpItem](https://github.com/DrachenkoAnastasiia/sdl_hmi_integration_guidelines/blob/develop/docs/Common/Structs/index.md#vrhelpitem)|false|array: true<br>minsize: 1<br>maxsize: 100|
 |menuTitle|String|false|maxlength: 500|
-|menuIcon|[Common.Image](../../common/structs/#image)|false||
-|keyboardProperties|[Common.KeyboardProperties](../../common/structs/#keyboardproperties)|false||
-|appID|Integer|true||
+|menuIcon|[Common.Image](https://github.com/DrachenkoAnastasiia/sdl_hmi_integration_guidelines/blob/develop/docs/Common/Structs/index.md#image)|false|-|
+|keyboardProperties|[Common.KeyboardProperties](https://github.com/DrachenkoAnastasiia/sdl_hmi_integration_guidelines/blob/develop/docs/Common/Structs/index.md#keyboardproperties)|false|-|
+|appID|Integer|true|-|
 
 ### Response
+|Result |Description |Message type WebSocket|Message type D-Bus|Message Params|
+|:------|:-----------|:---------------------|:-----------------|:-------------|
+|Success|SUCCESS: HMI has set the requested properties.|JSON response|Method return|code: 0|
+|Failure|INVALID_ID:appID is not valid (e.g.does not exist)|JSON response|Method return|code: 13|
+|Failure|INVALID_DATA: The data sent is invalid (invalid JSON syntax or parameters out of bounds or of wrong type)|JSON response|Method return|code: 11|
+|Failure|GENERIC_ERROR: The unknown issue occurred or other codes are not applicable.|JSON response|Method return|code: 22|
+_**Note**: In case HMI does not respond SDL's request during SDL-default timeout (10 sec), SDL will return GENERIC_ERROR  result code to the corresponding mobile app's request. Please see [Result Enumeration] for all SDL-supported codes._
```

Comment:
This note is showing up inside of the parameters table when looking at the preview: https://smartdevicelink.com/en/docs/pull_request/f1f00a3259aa9d8d21e0a221f913843f2575690e/ui/setglobalproperties/